### PR TITLE
Scaffold the project with @loopback/cli

### PR DESCRIPTION
**Pull request by raymondfeng**
Scaffold the project with `@loopback-cli`:
```
npm i -g @loopback/cli
lb4 extension
```

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,5 +1,4 @@
-Copyright (c) IBM Corp. 2017. All Rights Reserved.
```
Comment (dhmlau) [inline LICENSE]: Do we want this line too?

**Comment by dhmlau** [comment]
@raymondfeng , a quick question.. are we planning to put all our extensions in the monorepo?

**Comment by raymondfeng** [comment]
> are we planning to put all our extensions in the monorepo?

No.

File: LICENSE (hunk 1 above)
Comment (raymondfeng) [reply LICENSE]: Fixed.

File: LICENSE [hunk 2]
```diff
@@ -1,5 +1,4 @@
...
-Node module: loopback4-extension-starter
+Node module: @loopback/mqtt
```
Comment (dhmlau) [inline LICENSE]: Got your response on the monorepo.  I guess my question was originated from this line.  
In this case, would it be `Node module: loopback4-extension-mqtt`

**Review by dhmlau — APPROVED** [review]

**Pull request merged**